host-configs/rzwhippet-toss_4_x86_64_ib-clang@14.0.6.cmake

```diff
@@ -85,15 +85,17 @@ set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.23-hcrkmapza245iusfl4tva6a66vlynnwp" CACHE PA
 
 set(LUA_DIR "/usr" CACHE PATH "")
 
-set(RAJA_DIR "${TPL_ROOT}/raja-2024.07.0-hmdop6drvljlgj4433by42d4ugesd4zt" CACHE PATH "")
+set(RAJA_DIR "${TPL_ROOT}/raja-2024.07.0-ljyjfxo5jytxqglvczaf46yhbmr3gfc3" CACHE PATH "")
 
-set(UMPIRE_DIR "${TPL_ROOT}/umpire-2024.07.0-4bdhkgxmckkcfnhfv76gutbzn7gzrcum" CACHE PATH "")
+set(UMPIRE_DIR "${TPL_ROOT}/umpire-2024.07.0-et2uff5thiprgch43uftatndduzssxay" CACHE PATH "")
+
+set(OPENCASCADE_DIR "${TPL_ROOT}/opencascade-7.8.1-epyyoyevoozkzqoxdqxku2q4anwlqw52" CACHE PATH "")
```

Comment:
I tested configs on all platforms w/ `opencascade`, but only enabled it on a single toss4 config on LC.
I also enabled opencascade in Windows configs.